internal/datanode/binlog_io.go

```diff
@@ -279,7 +279,7 @@ func (b *binlogIO) genInsertBlobs(data *InsertData, partID, segID UniqueID, meta
 		fID, _ := strconv.ParseInt(blob.GetKey(), 10, 64)
 
 		k := JoinIDPath(meta.GetID(), partID, segID, fID, <-generator)
-		key := path.Join(Params.DataNodeCfg.StatsBinlogRootPath, k)
+		key := path.Join(b.ChunkManager.RootPath(), common.SegmentStatslogPath, k)
 
```

Comment:
change `Params.DataNodeCfg.StatsBinlogRootPath` to `common.SegmentStatslogPath` ?